Flowers.cpp

```diff
@@ -1,17 +1,60 @@
 #include <iostream>
 #include <string>
-#include "Flowers.h"
-#include "Flowersshop.h"
 #include "Bouquet.h"
 
-int main() {
+Flowers();
+Flowers(string name, int height, int size, int price, int quantity, int deliveryRate, string color);
+~Flowers() {}
 
-	Flowersshop shop(10);
-	shop.addflowers();
-	shop.sortprice();
-	shop.removeflowers(0);
-	shop.data();
-	Bouquet bouquet(15,shop.getassortment());
-	bouquet.databouquet();
+string getname()
+{
+    return name;
+}
+int getheight()
+{
+    return height;
+}
+int getsize()
+{
+    return size;
+}
+int getprice()
+{
+    return price;
+}
+int getquantity()
+{
+    return quantity;
+}
+int getdeliveryRate()
+{
+    return deliveryRate;
+}
+string getcolor()
+{
+    return color;
```

Comment:
Please use camelCase for function naming

Reply:
Fixed.